includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -1924,13 +1916,11 @@ static function() {
 
 				$styles = file_get_contents( $dark_mode_css_file ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
 
-				// Restrict rules to only when the user has requested the system use a dark color theme.
-				$new_styles = str_replace( '@media only screen', '@media only screen and (prefers-color-scheme: dark)', $styles );
 				// Allow for rules to override the light theme related rules.
-				$new_styles = str_replace( '.is-dark-theme.is-dark-theme', ':root', $new_styles );
-				$new_styles = str_replace( '.respect-color-scheme-preference.is-dark-theme body', '.respect-color-scheme-preference:not(._) body', $new_styles );
+				$styles = str_replace( '.is-dark-theme.is-dark-theme', 'body.is-dark-theme', $styles );
```

Comment:
> Do we have to change it to body.is-dark-theme?

It will surely work with `.is-dark-theme.is-dark-theme` but now we don't have `.is-dark-theme` selector in the document element so I replaced it with `body.is-dark-theme` to avoid confusion that from where `.is-dark-theme` selector is coming in `.is-dark-theme.is-dark-theme`. Should I remove this line or keep it as it is? Please share your thoughts on it.

Reply:
I'm not sure if I get the point. Since `.is-dark-theme.is-dark-theme` is a more general selector than `body.is-dark-theme` (yet having higher [specificity](https://css-tricks.com/specifics-on-css-specificity/), 20 vs. 11) I guess we should be good to keep it as is.